src/lib/builders/date-field/_internal/helpers.ts

```diff
@@ -430,7 +435,7 @@ export function setDescription(id: string, formatter: Formatter, value: DateValu
  */
 export function removeDescriptionElement(id: string) {
 	if (!isBrowser) return;
-	const el = document.getElementById(id);
+	const el = getElementById(id);
```

Comment:
This needs the rooteElement passed into it. And as a result so does this function.